packages/commerce-sdk-react/README.md

```diff
@@ -321,7 +321,56 @@ const Example = ({basketId}) => {
 }
 ```
 
-You could also import the mutation options as a constant like:
+#### Custom Endpoints and Dynamic `body` Mutate Parameters
```

Comment:
I asked for chatgpt's help... here is what it says:

```markdown
## `useCustomMutation`

The `useCustomMutation` hook facilitates communication with the SCAPI custom endpoint. It has a different signature than the other declared mutation hooks.

### Parameters

- `options` (Object): Configuration for the API request.
  - `method` (String): The HTTP method to use (e.g., 'POST', 'GET').
  - `customApiPathParameters` (Object): Contains parameters to define the API path.
    - `endpointPath` (String): Specific endpoint path to target in the API.
    - `apiName` (String): The name of the API.

- `clientConfig` (Object): Configuration settings for the client.
  - `parameters` (Object): Essential parameters required by the Salesforce Commerce Cloud API.
    - `clientId` (String): Your client ID.
    - `siteId` (String): Your site ID.
    - `organizationId` (String): Your organization ID.
    - `shortCode` (String): Short code for your organization.
  - `proxy` (String): Proxy address for API calls.

- `rawResponse` (Boolean): Determines whether to receive the raw response from the API or a parsed version.

### Usage

Below is a sample usage of the `useCustomMutation` hook within a React component:

```jsx
const clientConfig = {
    parameters: {
        clientId: 'CLIENT_ID',
        siteId: 'SITE_ID',
        organizationId: 'ORG_ID',
        shortCode: 'SHORT_CODE'
    },
    proxy: 'http://localhost:8888/mobify/proxy/api'
};

const mutation = useCustomMutation({
    options: {
        method: 'POST',
        customApiPathParameters: {
            endpointPath: 'test-hello-world',
            apiName: 'hello-world'
        }
    },
    clientConfig,
    rawResponse: false
});

// In your React component
<button onClick={() => mutation.mutate({ body: { test: '123' }})}>
    Send Request
</button>

```

Reply:
@kevinxh @joeluong-sfcc I've improved the documentation to cover the params, per the proposed suggestion. I've also kept the dynamic example I had because I think it's important to demonstrate a realistic use case where one might need to use this functionality.